java/vector/pom.xml

```diff
@@ -125,12 +124,12 @@
             <goals>
               <goal>test</goal>
             </goals>
-            <configuration>
+            <!-- <configuration>
               <classpathDependencyExcludes>
                 <classpathDependencyExclude>org.apache.arrow:arrow-memory-netty</classpathDependencyExclude>
               </classpathDependencyExcludes>
               <reportNameSuffix>netty</reportNameSuffix>
-            </configuration>
+            </configuration> -->
```

Comment:
Commented block, can we remove that or do we keep it ?

Reply:
I have removed it as it will not be used https://github.com/sirensolutions/arrow/pull/35/commits/cfd965270c0af9dda02e963f5e5711988d4633d9